cardano-api/src/Cardano/Api/Protocol/Byron.hs

```diff
@@ -8,52 +8,28 @@
 
 module Cardano.Api.Protocol.Byron
   (
-    -- * Protocol exposing the specific type
-    -- | Use this when you need the specific instance
-    mkConsensusProtocolByron
-
-    -- * Protocols hiding the specific type
-    -- | Use this when you want to handle protocols generically
-  , mkSomeConsensusProtocolByron
-
     -- * Client support
-  , mkNodeClientProtocolByron
+    mkNodeClientProtocolByron
   , mkSomeNodeClientProtocolByron
 
     -- * Errors
   , ByronProtocolInstantiationError(..)
   , renderByronProtocolInstantiationError
```

Comment:
These error types should move too. They are only used by the node-side support. The client side does not use them.